lib/file/index.js

```diff
@@ -10,8 +10,8 @@ function File(options) {
 	this._batches = [];
 
 	// Allow the batch header/control defaults to be overriden if provided
-	this.header = options.header ? _.merge(options.header, require('./header')(), _.defaults) : require('./header')();
-	this.control = options.control ? _.merge(options.header, require('./control'), _.defaults) : _.cloneDeep(require('./control'));
+	this.header = options.header ? _.defaultsDeep({}, options.header, require('./header')()) : require('./header')();
```

Comment:
passing an empty object as the first argument to defaultsDeep ensures it creates a new object for these settings instead of mutating `options.header`.